src/helpers/augmentedReality/storageNameCreator.js

```diff
@@ -1,9 +1,14 @@
 import { documentDirectory } from 'expo-file-system';
 
+import { consts } from '../../config';
+
+const { IMAGE_TYPE_REGEX } = consts;
+
 export const storageNameCreator = ({ dataItem, objectItem }) => {
+  const imageType = IMAGE_TYPE_REGEX.exec(objectItem.uri);
   const objectItemTitleWithoutSpaces = dataItem.title.replace(/\s+/g, '');
   const dataDirectoryName = `${objectItemTitleWithoutSpaces}_${dataItem.id}`;
-  const objectName = `${objectItem.title}.${objectItem.type}`;
+  const objectName = `${objectItem.title}.${imageType ? imageType[1] : objectItem.type}`;
```

Comment:
This only varies according to the texture types. For different types we can use the information from the server. For example, I am writing a few different file types:

Ch17_1001.**texture** = Ch17_1001.**png**
Ch17_1002.**texture** = Ch17_1002.**jpg**
augmentedReality.**target** = augmentedReality.**png**
Musik.**mp3** = Musik.**mp3**
test.**vrx** = test.**vrx**

Reply:
ok, this is not very clear to me when reading the line. can you refactor this to a function with a certain name and a comment including these example so that we still know about it in the future?

Reply:
I will do it. 👍

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/608/commits/afa7f49f3c7659b48f0bc217d6e549966e443c24